Mathlib/AlgebraicGeometry/Liouville.lean

```diff
@@ -0,0 +1,69 @@
+/-
+Copyright (c) 2025 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.AlgebraicGeometry.Morphisms.Proper
+
+/-!
+# Liouville's theorem
+
+In this file, we show that the global sections of a variety (integral scheme universally-closed
+and of finite type over `Spec K`) form a finite field extension of the base field.
```

Comment:
"Universally closed" isn't part of the definition of "variety" (while "integral" and "finite type" are), so I suggest something like
"if `X` is a variety over a field `K` (i.e. an integral scheme of finite type over `Spec K`), and `X` is universally closed, then the global sections of the structure sheaf of `X` are a finite field extension of `K`."